cmd/data.go

```diff
@@ -116,7 +116,8 @@ func (cmd *DataCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...interface
 		banner string
 	)
 	// Populate migration request id and migration type in conv object.
-	conv.Audit.MigrationRequestId = "SMT-" + uuid.New().String()
+	conv.Audit.MigrationRequestId, _ = utils.GenerateName("smt-job")
+	conv.Audit.MigrationRequestId = strings.Replace(conv.Audit.MigrationRequestId, "_", "-", -1)
```

Comment:
why do we need to do this replacement ? Can we not just edit the job id to comply with this structure if required ?

Reply:
`utils.GenerateName` is used very extensively through the code base, and requires this replacement to work. I have opted for not refactoring that other method currently.